bagisto/bagisto
# Admin is not receiving mail for customer registration, after enabling the feature from Notifications in Configure Section - 8893

## Issue Reference

#8893 

## Description


- Registration emails are now being sent when the setting is enabled in the backend.
- To send registration emails to customers, please enable the 'Send a confirmation e-mail after customer registration' option.
- To send registration emails to the admin, please enable the 'Send a confirmation e-mail to admin after customer registration' option.

## How To Test This?


- Navigate to the Admin panel, then go to Configure, Notifications, and select the third option: "Send a confirmation email to admin after customer registration."
- Enable this feature to send customer registration emails to the admin.
- And select the second option: "Send a confirmation email after customer registration."
- Enable this feature to send customer registration emails to the new customer.
- Return to the shop, create a new customer account, and check email of new customer and admin for the confirmation mail.

## Documentation
